projects/igniteui-angular/src/lib/chips/chip.component.ts

```diff
@@ -165,12 +165,24 @@ export class IgxChipComponent extends DisplayDensityBase {
      * ```html
      * <igx-chip #myChip [id]="'igx-chip-1'" [selectable]="true" [selected]="true">
      * ```
+     *
+     * Two-way data binding:
+     * ```html
+     * <igx-chip #myChip [id]="'igx-chip-1'" [selectable]="true" [(selected)]="model.isSelected">
+     * ```
      */
     @Input()
     public set selected(newValue: boolean) {
         this.changeSelection(newValue);
+        this.selectedChange.emit(this._selected);
```

Comment:
@MayaKirova This is not working if I have a selectable chip and change its state from the UI.
Here is a code:
```
<igx-chip [selectable]="true" [(selected)]="isSelected" [selectIcon]="mySelectIcon">
	With a custom select icon.
</igx-chip>
<span>Is Selected: {{isSelected}}</span>
```